### ISSUE TITLE
Eliminate iOS input zooming

### ISSUE THREAD

**Issue by brettinternet** [toboshii/hajimari#89]
**TL;DR**: If mobile font-sizes in text inputs are at least `16px`, browsers on iOS devices won't zoom into the element when the user selects the input.

---

**Is your feature request related to a problem? Please describe.**
Selecting (or "focusing" on) the search input on iOS devices makes the browser zoom into the element.

**Describe the solution you'd like**
Change the fontsize of the input to a value >= `16px`.

Since the search input's fontsize is `1rem`, I suppose you could also change the body element's fontsize to `16px` for the same result.

While I'm here, you may also consider setting border-radius to 0 since some browsers add default styles to the input, or `-webkit-appearance: none;` might do the trick.

**Describe alternatives you've considered**
None

**Additional context**
Submitting this as a feature rather than a bug because it's a rather annoying nuance with iOS devices.

For more context, here's a similar submission I made to Meta StackOverflow: https://meta.stackoverflow.com/q/389464/6817437
